tests/examples/example1.out

```diff
@@ -66,9 +66,9 @@ Simulation Metadata
       Well seperation: fracture height:                     900.00 meter
       Fracture area:                                     810000.00 m**2
       Reservoir volume:                              1000000000 m**3
-      Reservoir hydrostatic pressure:                       100.00 kPa
```

Comment:
Note previous examples all had incorrect default value of 100.00 kPa, examples are now updated with calculated hydrostatic pressure, and default is now ~29k kPa